src/tof/chopper.py

```diff
@@ -18,15 +18,16 @@ class Chopper:
     Parameters
     ----------
     frequency:
-        The frequency of the chopper.
+        The frequency of the chopper. Must be positive.
     open:
         The opening angles of the chopper cutouts.
     close:
         The closing angles of the chopper cutouts.
     distance:
         The distance from the source to the chopper.
     phase:
-        The phase of the chopper.
+        The phase of the chopper. The phase offset is applied in the opposite direction
+        to the chopper rotation direction.
```

Comment:
Can you clarify this a bit more? Do I read this correctly that the meaning of phase changes when we change the rotation direction?